app/Http/Controllers/RequestTurnoverController.php

```
@@ -91,39 +91,67 @@ public function outgoing(Request $request, int $warehouse)
+                $remarks = $item['remarks'] ?? '';
+                if ($remarks === 'Request to Transfer') {
+                    $hasTransfer = true;
+                    $finalRemarks = 'Request to Transfer';
+                } elseif ($remarks === 'Request to Return') {
+                    $hasReturn = true;
+                    $finalRemarks = 'Request to Return';
+                } elseif ($remarks === 'Others') {
+                    $finalRemarks = $item['remarks_other'];
+                } else {
+                    $finalRemarks = $remarks ?: null;
+                }
```

PHPMD (2.15.0)

The method store uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)